README

```diff
@@ -0,0 +1,28 @@
+===
+
+ This project is aimed to provide an Attester implementation
+ for Attestation process which also should include local (and remote)
+ plaform provisioning.
+
+ As for attestation itself the Fobnail/charra (https://github.com/fobnail/charra)
+ project could be used as an example and the base of future extentions.
+ It provides a simple (not full) interaction between Attester and Verifier
+ over CHARRA protocol.
+
+ But local (and remote) provisioning must be performed before the attestation
+ starts.
+
+===
+ The project depends on libcoap library.
+ Install libcoap:
+ - git clone --depth=1 --recursive -b 'develop' 'https://github.com/obgm/libcoap.git'
+ - cd libcoap/
```

Comment:
@0xDen Please say exactly which commit.

Reply:
I don't think we should specify any commit sha directly. I believe it would be better to wait when the development features will be merged into one of the stable branches... I used 'develop' branch since there are still no such functionality in stabe branches... But if commit_sha is still necessary I will add them